### PR TITLE
Add persistence for webhooks

### PR DESCRIPTION
**Pull request by ChrisSchinnerl**

### CODE REVIEW

File: stores/webhooks.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	"go.sia.tech/renterd/webhooks"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+type (
+	dbWebhook struct {
+		Model
+
+		Module string `gorm:"uniqueIndex:idx_url_module_event;NOT NULL;size:255"`
```
Comment (peterjan) [inline stores/webhooks.go:13]: would've gone for `idx_module_event_url`

File: internal/node/node.go [hunk 2]
```diff
@@ -253,15 +253,21 @@ func NewBus(cfg BusConfig, dir string, seed types.PrivateKey, l *zap.Logger) (ht
 		dbConn = stores.NewSQLiteConnection(filepath.Join(dbDir, "db.sqlite"))
 	}
 
-	hooksMgr := webhooks.NewManager(l.Named("webhooks").Sugar())
-	alertsMgr := alerts.NewManager(hooksMgr)
+	alertsMgr := alerts.NewManager()
```
Comment (peterjan) [inline internal/node/node.go:256]: Would it be slightly cleaner to pass a pointer to a broadcaster in the constructor and default to the NoOp broadcaster if it's nil? That avoids `RegisterWebhookBroadcaster` and the `panic` we have to throw.

File: stores/migrations.go [hunk 3]
```diff
@@ -40,6 +40,9 @@ var (
 
 		// bus.AutopilotStore tables
 		&dbAutopilot{},
+
+		// webhooks.Store tables
```
Comment (peterjan) [inline stores/migrations.go:44]:
```suggestion
		// webhooks.WebhookStore tables
```

File: stores/sql.go [hunk 4]
```diff
@@ -414,6 +414,9 @@ func (s *SQLStore) retryTransaction(fc func(tx *gorm.DB) error, opts ...*sql.TxO
 		return false
 	}
 	var err error
+	if false {
+		panic("trigger git conflict to add ErrRecordExists to exception")
```
Comment (peterjan) [inline stores/sql.go:418]: wait wut?

**Review by peterjan — APPROVED** [review]
left some comments but overall looks good

File: internal/node/node.go (hunk 2 above)
Comment (ChrisSchinnerl) [reply internal/node/node.go:256]: We can't do that unfortunately. Because of the dependencies the database, alert manager and broadcaster have.

**Pull request merged**